common/lib/capa/capa/safe_exec/exceptions.py

```diff
@@ -0,0 +1,9 @@
+"""
+Exceptions related to safe exec.
+"""
+
+class CodejailServiceUnavailable(Exception):
+    """
+    An exception that is raised whenever Codejail service is unavailable.
+    """
+    pass  # lint-amnesty, pylint: disable=unnecessary-pass
```

Comment:
why is this necessary?

Reply:
@mariajgrimaldi We are basically copying the edxnotes structure https://github.com/edx/edx-platform/blob/master/lms/djangoapps/edxnotes/exceptions.py

Reply:
These `lint-amnesty` comments were added by a tool that allowed us to ignore existing lint failures and prevent new ones from coming in -- if you're copying code from somewhere, you'll want to resolve the lint failure instead.

In this case, you can actually get rid of the entire line, both `pass` and linter comment. The `pass` is apparently unneeded because the comment serves as the body of the class (this surprised me, but OK!) and that's what the linter was complaining about. I suspect that when the original code was written in notes, the comments weren't there at first, so the author added `pass` to wrap up the class and then forgot to remove it later. :-)